src/olympia/abuse/cinder.py

```diff
@@ -288,12 +288,11 @@ def get_extended_attributes(self):
             data['release_notes'] = self.get_str(
                 self.addon.current_version.release_notes
             )
-        # In addition, the URL/email fields can't be sent if blank as they
-        # would not be considered valid by Cinder.
-        for field in ('homepage', 'support_email', 'support_url'):
-            if value := getattr(self.addon, field):
-                data[field] = self.get_str(value)
-
+        # The URL/email fields can't be sent as empty strings as they would not
+        # be considered valid by Cinder.
+        data['homepage'] = self.get_str(self.addon.homepage) or None
```

Comment:
should the `CinderUser` homepage also be handled the same?

Reply:
Good call. Looks like we also need a migration there!

Reply:
And while fixing the API for users I also discovered the add-ons API wasn't validating properly (it allows URLs even if they don't have an http/https protocol) so I'll be fixing that as well...

Reply:
https://github.com/mozilla/addons-server/pull/21573/commits/069d04965ffc25d9ef0d4495ae1461e6444cdedf and https://github.com/mozilla/addons-server/pull/21573/commits/d35d776495f1886f8bfe2e87c1e89ad488fc7725